### DISCUSSION TITLE
Resolver v3.0 is now on the 'master' branch where development and support will continue

### DISCUSSION THREAD

**Discussion by nicklansley** [gs1/GS1_DigitalLink_Resolver_CE#96] [Announcements]
**Resolver CE v3.0 has moved from the 'V3' branch to the 'master' branch.**

It's now time to point your code branch back to the 'master' branch to keep up with the latest updates and improvements as work on v3.0 continues. Branch 'V3' is no longer maintained directly. 

We've also stopped development and maintenance on version 2.6, but you can still find the code in the 'v2.6' branch of this repository: https://github.com/gs1/GS1_DigitalLink_Resolver_CE/tree/v2.6

We recommend that you upgrade to version 3.0 to take advantage of the new features, simplified services, and many improvements. We are looking forward to your feedback and contributions to the project.

**Comment by Amambicyprain** [comment]
I am not sure if this is the right place to ask my questions but i am stuck with GS1_DIGITALlINK RESOLVER_CE. for quite some time now. i am trying to deploy to a digital ocean ubuntu 20.04 droplet. i am able to deploy but when i try to access it get a 502 error . I will be glad if anyone can help me out
